src/python/zquantum/qaoa/maxcut.py

```diff
@@ -11,21 +11,25 @@
 
 
 def create_farhi_qaoa_circuits(
-    hamiltonians: List[QubitOperator], number_of_layers: int
+    hamiltonians: List[QubitOperator], number_of_layers: Union[int, List[int]]
 ):
-    """Creates parameterizable quantum circuits based on the farhi qaoa ansatz for each 
-    hamiltonian in the input list using the set number of layers. 
+    """Creates parameterizable quantum circuits based on the farhi qaoa ansatz for each
+    hamiltonian in the input list using the set number of layers.
 
     Args:
         hamiltonians (List[QubitOperator]): List of hamiltonians for constructing the
             circuits
         number_of_layers (int): The number of layers of the ansatz in the circuit
```

Comment:
I'd just add information about how `number_of_layers` works – i.e. what happens if `int` is provided and what happens if `list` is provided.

Reply:
Addressed here: 1c099c0